package.json

```diff
@@ -101,7 +101,7 @@
 
     "protractor": "^3.2.2",
 
-    "parse5": "^1.5.1",
+    "parse5": "^2.1.5",
```

Comment:
someone added this for universal support which only supports 1.x.x it really shouldn't be in the project tbh because it's only needed if someone installs universal